src/main/java/de/tum/in/www1/artemis/service/connectors/pyris/dto/lectureingestionwebhook/PyrisLectureUnitsStatusUpdate.java

```diff
@@ -0,0 +1,13 @@
+package de.tum.in.www1.artemis.service.connectors.pyris.dto.lectureingestionwebhook;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+
+/**
+ * Represents a webhook data transfer object for lecture units in the Pyris system.
+ * This DTO is used to encapsulate the information related to updates of lecture units,
+ * providing necessary details such as lecture and course identifiers, names, and descriptions.
+ */
+@JsonInclude(JsonInclude.Include.NON_EMPTY)
+
+public record PyrisLectureUnitsStatusUpdate(Long lectureUnitId, Long lectureId, Long courseId) {
```

Comment:
```suggestion
public record PyrisLectureUnitsStatusUpdateDTO(Long lectureUnitId, Long lectureId, Long courseId) {
```
If this is a DTO (which it is according to the comment) it should also have it in its name